### ISSUE TITLE
Allow setting configuration using UEFI variables

### ISSUE THREAD

**Issue by sftim** [EmilyShepherd/kiOS#19]
_As a cluster operator
I want to set a UEFI variable to control the DNS resolver configuration in Kios
So that I can use custom DNS settings
And I do not need to bake a new image for each different set of DNS servers._

That, basically. Some things need to be right very early in boot, and DNS is often one of those things.

**Comment by EmilyShepherd** [comment]
Agreed - I think it would be sensible to have some better form of UEFI variable support. I _suspect_ I'll add this support via an optional bootstrap pod (a la kios-aws' bootstrap container) but open to other suggestions
